SuperBuild/External_LZMA.cmake

```diff
@@ -0,0 +1,96 @@
+
+set(proj LZMA)
+
+# Set dependency list
+set(${proj}_DEPENDENCIES "")
+
+# Include dependent projects if any
+ExternalProject_Include_Dependencies(${proj} PROJECT_VAR proj DEPENDS_VAR ${proj}_DEPENDENCIES)
+
+if(Slicer_USE_SYSTEM_${proj})
+  unset(LZMA_INCLUDE_DIR CACHE)
+  unset(LZMA_LIBRARY CACHE)
+  find_path(LZMA_INCLUDE_PATH lzma.h)
+  find_library(LZMA_LIBRARY lzma)
+endif()
+
+# Sanity checks
+if(DEFINED LZMA_INCLUDE_DIR AND NOT EXISTS ${LZMA_INCLUDE_DIR})
+  message(FATAL_ERROR "LZMA_INCLUDE_DIR variable is defined but corresponds to nonexistent directory")
+endif()
+if(DEFINED LZMA_LIBRARY AND NOT EXISTS ${LZMA_LIBRARY})
+  message(FATAL_ERROR "LZMA_LIBRARY variable is defined but corresponds to nonexistent directory")
+endif()
+
+if((NOT DEFINED LZMA_INCLUDE_DIR
+   OR NOT DEFINED LZMA_LIBRARY) AND NOT Slicer_USE_SYSTEM_${proj})
+
+  ExternalProject_SetIfNotDefined(
+    Slicer_${proj}_GIT_REPOSITORY
+    "${EP_GIT_PROTOCOL}://github.com/Slicer/lib_lzma.git"
+    QUIET
+    )
+
+  # Use v5.2.2 to satisfy CPython 3.7 Windows build requirements
+  # See https://github.com/python/cpython/blob/3.7/PCbuild/readme.txt
```

Comment:
https://github.com/python/cpython/blob/v3.6.7/PCbuild/readme.txt
I think you want to specify for 3.6.7 which is the version we currently use.  Technically 3.7 also uses v5.2.2 so technically the comment isn't wrong, but not the version we are using.

Reply:
Good catch :pray: